src/components/ShoppingListItem/ShoppingListItem.js

```diff
@@ -86,23 +90,30 @@ export function ShoppingListItem({
         className={classNames(
           'ShoppingListItem',
           { 'ShoppingListItem--checked': item.isChecked },
-          'aisle'
+          'aisle',
+          'padded'
         )}
         onClick={onClick}
         type="button"
       >
         {item.aisleLocation && item.aisleLocation.aisleNo}
       </button>
       <div className="ShoppingListItem-buttons">
+        <div>
+          <i
+            class="fas fa-arrow-up"
```

Comment:
should be `className`

Reply:
argh! the copy paste gets me always! I got it thanks!